inscriptions/ISic003523.xml

```diff
@@ -76,33 +80,41 @@
                     <physDesc>
                         <objectDesc>
                             <supportDesc>
-                                <support>
+                                <support><p>An entire but eroded slab of marble with the text fully preserved.</p>
                                     <material ana="#material.stone.marble" ref="http://www.eagle-network.eu/voc/material/lod/48.html">marble</material>
                                     <objectType ana="#object.plaque" ref="http://www.eagle-network.eu/voc/objtyp/lod/259">plaque</objectType>
-                                    <dimensions><!--Default values and requires editing-->
-                                        <height unit="cm"/>
-                                        <width unit="cm"/>
-                                        <depth unit="cm"/>
+                                    <dimensions><!--from IMusMessina 16-->
+                                        <height unit="cm">23</height>
+                                        <width unit="cm">23</width>
+                                        <depth unit="cm">1,8-2</depth>
```

Comment:
NB we do not use the continental comma for decimal division, but the period, so 1.8-2